cosipy/response/FullDetectorResponse.py

```diff
@@ -190,21 +190,24 @@ def _open_rsp(cls, filename, Spectrumfile=None,norm="Linear" ,single_pixel = Fal
                 elif key == 'SA':
                     area_sim = float(line[1])
                     
-                elif key == "SP" and line[1]!="true" and line[1]!="false" :
-                    norm = str(line[1])
+                elif key == "SP" :
                     
+                    try :
```

Comment:
If I understand this correctly, this is mostly so it is backwards compatible? Why do some files have an empty SP and some don't?

Reply:
I think Andreas said he used a new(old) version or maybe this is a bug. Anyway this line is more to check if it was empty or not and warn the user

Reply:
Ok, thanks.